go/libkb/output_windows.go

```diff
@@ -90,8 +92,20 @@ type ColorWriter struct {
 
 // Rough emulation of Ansi terminal codes.
 // Parse the string, pick out the codes, and convert to
-// calls to SetConsoleTextAttribute
+// calls to SetConsoleTextAttribute.
+//
+// This function must mix calls to SetConsoleTextAttribute
+// with separate Write() calls, so to prevent pieces of colored
+// strings from being interleaved by unsynchronized goroutines,
+// we unfortunately must lock a mutex.
+//
+// Notice this is only necessary for what is now called
+// "legacy" terminal mode:
+// https://technet.microsoft.com/en-us/library/mt427362.aspx?f=255&MSPPError=-2147217396
 func (cw *ColorWriter) Write(p []byte) (n int, err error) {
+	mutex.Lock()
+	defer mutex.Unlock()
```

Comment:
oh yeah, that sucks, but since stdout and stderr seem to have multiple ColorWriter instances each, a per-object lock didn't work. Let me try it again.